Makefile

```diff
@@ -53,7 +53,7 @@ package_tu_collector: build_tu_collector package_dir_structure
 	cd $(CC_BUILD_DIR) && \
 	ln -sf ../lib/python2.7/tu_collector/tu_collector.py bin/tu_collector
 
-package: package_dir_structure package_plist_to_html package_tu_collector
+package: package_dir_structure set_git_commit_template package_plist_to_html package_tu_collector
```

Comment:
Do we need to set this config value at each package build?

Reply:
There are lot of places where we can set this: `analyzer`, `web`, `plist_to_html`, `tu_collector`, `warning-to-plist converter`, `build-logger` etc. These packages can be built separately.

I think it is enough that we set this config value when somebody tries to build the main package.